docs/resources/aws_iam_access_keys.md

```diff
@@ -0,0 +1,164 @@
+---
+title: About the aws_iam_access_keys Resource
+---
+
+# aws_iam_access_keys
+
+Use the `aws_iam_access_keys` InSpec audit resource to test properties of all or multiple IAM Access Keys.
+
+To test properties of a Access Key, use the `aws_iam_access_key` resource.  You may also test properties of the access keys of a particular user using the `aws_iam_user` resource.
+
+Access Keys are tightly coupled to the AWS User resources.  This resource is best used when you need so perform audits of all keys, or keys specified by criteria that don't relate to any particular user.  
+
+<br>
+
+## Syntax
+
+An `aws_iam_access_keys` resource block uses an optional filter to select a group of access keys and then tests that group.
+
+    # Do not allow any access keys
+    describe aws_iam_access_keys do
+      it { should_not exist }
+    end
+
+    # Don't let fred have access keys, using filter argument syntax
+    describe aws_iam_access_keys.where(username: 'fred') do
+      it { should_not exist }
+    end  
+
+    # Don't let fred have access keys, using filter block syntax (most flexible)
+    describe aws_iam_access_keys.where { username == 'fred' } do
+      it { should_not exist }
+    end    
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Disallow access keys created more than 90 days ago
+
+    describe aws_iam_access_keys.where { created_age > 90 } do
+      it { should_not exist }
+    end 
+
+<br>
+
+## Matchers
+
+### exists
+
+This matcher causes the control to pass if the filter returned at least one result. You can negate the meaning using should_not (that is, you expect no matches).
```

Comment:
The control will pass if the filter returns at least one result. Use should_not if you expect zero matches.